pkg/tf2pulumi/convert/tf12.go

```diff
@@ -1902,6 +1902,13 @@ var tf12builtins = map[string]*model.Function{
 				ReturnType: elementType,
 			}, diagnostics
 		})),
+	"sha1": model.NewFunction(model.StaticFunctionSignature{
+		Parameters: []model.Parameter{{
+			Name: "input string",
```

Comment:
How is this name used?  In all other cases here - it is a valid identifier, but this new one includes a space. Should this case also be a valid identifier, like `input`?

Reply:
Done.